NEWS.md

```diff
@@ -1,9 +1,14 @@
 ClimaLand.jl Release Notes
 ========================
 
-main
+Main
```

Comment:
I am not saying that we should not capitalize this, but `main` is a name of a git object (and `v0.14.4`, `v0.14.3`, ... too)